docs/quickstart/next-steps.rst

```diff
@@ -26,40 +26,31 @@ to see a summary of the TODOs again:
 
 Let's have a look at some of them one-by-one and explain what they are for:
 
-1. Creating a ``requirements.txt``: This file is used for pinning the Python
-   dependencies of your instance to specific versions in order to achieve
-   reproducible builds when deploying your instance. You can generate this file
-   in the following fashion (note, this is only for the *instance* and not
-   the *data model*):
-
-   .. code-block:: console
-
-      $ cd my-repository/
-      $ workon my-repository-venv
-      (my-repository-venv)$ pip install --editable .
-      (my-repository-venv)$ pip install pip-tools
-      (my-repository-venv)$ pip-compile
+# we may not need 1. if we use pipenv
```

Comment:
Shouldn't it make reference to what is "1"? Since the text about the step are deleted the user might be confused. Maybe something like:

The following steps use pipenv. Therefore, there is no need to create a ``requirements.txt`` file.

Reply:
that was only a comment, I've removed it